scripts/after_prepare.js

```diff
@@ -0,0 +1,85 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+const GLOBAL_FILE_MATCHER = /(.android|.ios|.windows|.wp8).\w+$/;
+const platformsData = {
+	android: {
+		nativePath: (projectRoot) => path.join(projectRoot, "platforms", "android", "assets"),
+		fileMatcher: /(.android).\w+$/
+	},
+	ios: {
+		nativePath: (projectRoot) => path.join(projectRoot, "platforms", "ios"),
+		fileMatcher: /(.ios).\w+$/
+	},
+	windows: {
+		nativePath: (projectRoot) => path.join(projectRoot, "platforms", "windows"),
+		fileMatcher: /(.windows).\w+$/
+	},
+	wp8: {
+		nativePath: (projectRoot) => path.join(projectRoot, "platforms", "wp8"),
+		fileMatcher: /(.wp8).\w+$/
+	},
+};
+
+module.exports = function (context) {
+	const projectRoot = context.opts.projectRoot;
+	const platforms = context.opts.platforms;
+
+	copyResources(platforms, projectRoot);
+}
+
+function copyResources(platforms, projectRoot) {
+	const excludePaths = [
+		path.join(projectRoot, "platforms"),
+		path.join(projectRoot, "node_modules"),
+		path.join(projectRoot, "plugins"),
+		path.join(projectRoot, "Plugins")
+	];
+
+	walkSync(projectRoot, (filePath) => {
+		if (GLOBAL_FILE_MATCHER.test(filePath)) {
+			const relativePath = filePath.substring(projectRoot.length + 1);
+			removeFromPlatform(projectRoot, relativePath);
+			platforms.forEach(platform => {
+				if (platformsData[platform].fileMatcher.test(filePath)) {
+					const dest = path.join(platformsData[platform].nativePath(projectRoot), path.dirname(relativePath), path.basename(filePath).replace(`.${platform}`, ""));
+					copyFile(filePath, dest);
+				}
+			});
+		}
+	}, (filePath) => {
+		return !!excludePaths.find(path => {
+			return filePath === path;
```

Comment:
this can be shortened to `return !!excludePaths.find(path => filePath === path);`